packages/react-components/components/PhoneNumberInput.tsx

```diff
@@ -3,12 +3,16 @@ import ValidatedTextInput, {
   PhoneValidatorProps,
 } from '@celo/react-components/components/ValidatedTextInput'
 import colors from '@celo/react-components/styles/colors'
+import SmsRetriever from '@celo/react-native-sms-retriever'
```

Comment:
Have you confirmed that this doesn't break the verifier? Because it doesn't have this package set up so I assume the import would fail. Please run the verifier and double check the phone input screen is okay